src/schema/v2/order/exchangeErrorHandling.ts

```diff
@@ -0,0 +1,28 @@
+import { ORDER_MUTATION_FLAGS } from "./sharedOrderTypes"
+
+type ExchangeError = Error & {
+  statusCode: number
+  body?: {
+    message: string
+    code: string
```

Comment:
I like how Stripe aims for the [message](https://docs.stripe.com/api/errors#errors-message) to be human readable and for end users.

> message nullable string
A human-readable message providing more details about the error. For card errors, these messages can be shown to your users.